iXBRLViewerPlugin/__init__.py

```diff
@@ -89,7 +89,7 @@ def generateViewer(
         showValidationMessages: bool = False,
         useStubViewer: bool = False,
         zipViewerOutput: bool = False,
-        features: Optional[list[str]] = None):
+        features: Optional[list] = None):
```

Comment:
It would be preferable to keep `list[str]` and make this work on Python 3.8 with `from __future__ import annotations`